forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for simulate_gps_data module"""
+
+import pytest
+
+from forest.bonsai.simulate_gps_data import get_path
+
+coords1 = (51.458498, -2.59638)
```

Comment:
You can still put this data in a fixture. I was only referring to the call to function under test in my previous comment.